src/status.js

```diff
@@ -0,0 +1,9 @@
+export function updateStatus(event, status) {
+  event.target.nextElementSibling.classList.toggle('completed');
+  status[event.target.dataset.id].completed = event.target.checked;
+  return status;
+}
+export function editTodo(event, editable) {
+  editable[event.target.dataset.index].description = event.target.value;
+  return editable;
+}
```

Comment:
- [ ] As per the learning objective, We have to use proper ES6 syntax. Kindly try to use the arrow function as per ES6 guidelines.

Reply:
fixed